pages/common/gist.md

```diff
@@ -2,6 +2,10 @@
 
 > Upload code to https://gist.github.com.
 
+- Authenticate gist on this computer:
```

Comment:
I'm not sure I understand this description. @agnivade, @sbrl, how did you interpret this sentence?

Reply:
It is from `gist --help`. Expect A more accurate description. What about
> Login in gist on this computer.

Reply:
@jasonxia23 Yeah, that's a good idea!

Reply:
I like that too :+1: